### DISCUSSION TITLE
Cannot add a classes into "Rich Text" Field like in WYSIWG that has option for code or visual

### DISCUSSION THREAD

**Discussion by esilverman682** [wpengine/atlas-content-modeler#596] [Ideas]
I need to add columns on the frontend which can easily be done by adding classes to the fields. ACF offers this and I can style it easily on the frontend. 

Currently populating the data with dangerously set html and field name
example:

     <main className="content content-single">
        <div className="wrap">
          <div dangerouslySetInnerHTML={{ __html: model?.bodyContent ?? '' }} />
        </div>
      </main>

I need to be able to manipulate the content inside to make columns etc.

 
 
### Version information
- Plugin version: latest
- WordPress version: 6.0.1
- Operating system: MAC
- Browser: Chrome

**Comment by nickcernis** [comment]
Thanks for your request, @esilverman682.

I can see the benefit of being able to switch from visual to code mode for rich text fields.

We'll leave this open so that we can let you know if we decide to add the feature.

**Comment by esilverman682** [comment]
ok thanks.
